dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageRenderUtil.java

```diff
@@ -228,143 +230,192 @@ private Container getWorkingContainerById(final String containerIdOrPath, final
-                        if (field != null && UtilMethods.isSet(field.values())) {
-                            widgetPreExecute.append(field.values());
-                        }
-                    }
+                    this.widgetPreExecute(contentlet);
+                    this.addAccrueTags(contentlet);
+
+                    if (personalizedContentlet.getPersonalization().equals(includeContentFor)) {
 
-                    // Check if we want to accrue the tags associated to each contentlet on
-                    // this page
-                    if (Config.getBooleanProperty("ACCRUE_TAGS_IN_CONTENTS_ON_PAGE", false)) {
+                        final Map<String, Object> contentPrintableMap = Try.of(() -> ContentletUtil.getContentPrintableMap(user, contentlet))
+                                .onFailure(f -> Logger.warn(this.getClass(), f.getMessage())).getOrNull();
+                        if (contentPrintableMap == null) {
 
-                        // Search for the tags associated to this contentlet inode
-                        final List<Tag> contentletFoundTags = this.tagAPI.getTagsByInode(contentlet.getInode());
-                        if (contentletFoundTags != null) {
-                            this.pageFoundTags.addAll(contentletFoundTags);
+                            continue;
                         }
-                    }
-                    
-                    if(personalizedContentlet.getPersonalization().equals(includeContentFor)) {
-                        final Map<String,Object> contentPrintableMap = Try.of(()-> ContentletUtil.getContentPrintableMap(user, contentlet)).onFailure(f-> Logger.warn(this.getClass(), f.getMessage())).getOrNull();
-                        if(contentPrintableMap==null)continue;
+
                         contentPrintableMap.put("contentType", contentlet.getContentType().variable());
                         personalizedContentletMap.add(contentPrintableMap);
+                        contentletIncludedCount++;
                     }
                 }
 
                 contentMaps.put(CONTAINER_UUID_PREFIX + uniqueUUIDForRender, personalizedContentletMap);
             }
-            for(Map.Entry<String,List<String>> entry : containerUuidPersona.entrySet()) {
+
+            for (final Map.Entry<String, List<String>> entry : containerUuidPersona.entrySet()) {
+
                 contextMap.put("contentletList" + entry.getKey(), entry.getValue());
-                contextMap.put("totalSize"  + entry.getKey(), entry.getValue().size());
+                contextMap.put("totalSize"      + entry.getKey(), entry.getValue().size());
             }
-            
 
             raws.add(new ContainerRaw(container, containerStructures, contentMaps));
         }
-        
+
         return raws;
     }
 
+    private boolean isLive(final HttpServletRequest request) {
+
+        return request != null && request.getSession(false) != null && request.getSession().getAttribute("tm_date") != null ?
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ternary operators that can be simplified with || or &&](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5316867)